lib/matplotlib/ticker.py

```diff
@@ -2386,23 +2382,14 @@ def tick_values(self, vmin, vmax):
         decades = np.arange(math.floor(log_vmin) - stride,
                             math.ceil(log_vmax) + 2 * stride, stride)
 
-        if hasattr(self, '_transform'):
```

Comment:
Does this never happen?

Reply:
Correct. Not in the last seven years or so.